tasks/eva_2156/deprecate_variant_from_variant_warehouse.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+
+import pymongo
+
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
+
+
+def deprecate(settings_xml_file, database_name, contigs=None):
+    """
+    Connect to mongodb and retrieve all variants that needs to be deprecated.
+    Copy the variant in the operation collection and delete them from the submitted variant collections.
+    """
+    with pymongo.MongoClient(get_mongo_uri_for_eva_profile('production', settings_xml_file)) as mongo_handle:
+        variant_collection = mongo_handle[database_name]['variants_2_0']
+        deleted_variant_collection = mongo_handle[database_name]['to_delete_variants_2_0']
+
+        cursor = variant_collection.find({'chr': {'$in': contigs}})
+        drop_statements = []
+        insert_statements = []
+        for variant in cursor:
+            insert_statements.append(pymongo.InsertOne(variant))
```

Comment:
Suggest adding a timestamp and a reason for deletion in the inserted variant document....

Reply:
Never mind... looks like the script has already been run...